edx_proctoring/api.py

```diff
@@ -640,9 +641,43 @@ def update_attempt_status(exam_id, user_id, to_status, raise_if_not_found=True,
         exam_attempt_obj.started_at = datetime.now(pytz.UTC)
         exam_attempt_obj.save()
 
+    if ProctoredExamStudentAttemptStatus.needs_status_change_email(exam_attempt_obj.status):
+        send_proctoring_attempt_status_email(exam_attempt_obj)
+
     return exam_attempt_obj.id
 
 
+def send_proctoring_attempt_status_email(exam_attempt_obj):
+    """
+    Sends an email about change in proctoring attempt status.
+    """
+
+    email_template = loader.get_template('emails/proctoring_attempt_status_email.html')
+    body = email_template.render(
+        Context({
+            'student_name': exam_attempt_obj.student_name,
+            'course_name': 'edx demo course',
+            'exam_name': exam_attempt_obj.proctored_exam.exam_name,
+            'status': exam_attempt_obj.status,
+            'date': exam_attempt_obj.modified
+        })
+    )
+
+    subject = (
+        'Status update of {exam_name} for {course_name}'.format(
```

Comment:
@chrisndodge @griffresch A single subject line for all states, or one for each?
Maybe something like Proctoring Session Results Update for {course_name} {exam_name}?